### PR TITLE
Profiler::benchmark() uses TimeSpan instead of double

### PR DESCRIPTION
**Pull request by christopherlagerhult**
review @marcusnaslund

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Well, that was easy. :)

File: source/base/Profiler.ooc [hunk 1]
```diff
@@ -50,17 +50,18 @@ Profiler: class {
 		This _profilers free()
 		This _profilers = null
 	}
-	benchmark: static func (task: Func, timeoutMilliseconds := 1000.0) -> Double {
+	benchmark: static func (task: Func, timeoutMilliseconds := TimeSpan milliseconds(1000.0)) -> Double {
```
Comment (marcusnaslund) [inline source/base/Profiler.ooc]: The name should just be `timeout` now.

Comment (christopherlagerhult) [inline source/base/Profiler.ooc]: oh right :)

File: source/base/Profiler.ooc [hunk 2]
```diff
@@ -50,17 +50,18 @@ Profiler: class {
...
+	benchmark: static func (task: Func, timeout := TimeSpan milliseconds(1000.0)) -> Double {
 		numberOfIterations := 0
 		timer := WallTimer new()
 		totalTime := TimeSpan new(0)
-		while (totalTime toMilliseconds() < timeoutMilliseconds) {
+		while (totalTime < timeout) {
 			numberOfIterations += 1
 			timer start()
 			task()
 			totalTime += timer stop()
```
Comment (marcusnaslund) [inline source/base/Profiler.ooc]: Does rock automatically infer that `+= x` means `= this + x`? @thomasfanell There doesn't seem to be a `+=` defined on `TimeSpan`.

Comment (thomasfanell) [inline source/base/Profiler.ooc]: Yes, `a += b` is unwrapped to `a = a + b`

**Pull request merged**